.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,6 @@
 ## Talk description
-(Please also name frameworks that you promote/use in your talk.)
+
+Please do not forget to drag & drop a teaser image here and also name frameworks that you promote/use in your talk.
```

Comment:
Maybe we should add some more information:

> A teaser image can for example be a graphic that summarizes your presentation or a portrait of you. With the upload you confirm that you have the rights to the image. You also allow us to use the image to visualize and promote your presentation, for example for Meetup, Facebook and Twitter).

Reply:
![giphy.gif](https://media.giphy.com/media/3o7ZeTmU77UlPyeR2w/giphy.gif)